pkgs/top-level/all-packages.nix

```diff
@@ -16763,23 +16762,21 @@ with pkgs;
 
   bump = callPackage ../development/tools/github/bump { };
 
-  libbpf_1 = callPackage ../os-specific/linux/libbpf { };
+  libbpf = callPackage ../os-specific/linux/libbpf { };
+  # backwards compatibility for a while, 2022-11
+  libbpf_1 = libbpf;
```

Comment:
```suggestion
```

I don't think we need t keep that alias if we remove all of them from the tree in this PR.

Reply:
I think that alias is used by @saschagrunert in external repos, so my intent was to remove it after one release (so after may-ish)
It would have been better with a deprecation warning attached but I'm not quite sure how to do that?

If you prefer just removing though I don't mind either way, I'm not using this myself.

Reply:
If it's a deprecated backward-compatibility alias, it should be in `aliases.nix`.

Reply:
I usually pin nixpkgs to a certain commit so I would see on upgrade that the package does not exist any more. Means I will not block on that change :)

Reply:
Didn't know about `aliases.nix`, thanks! Just moved it here.

And, well, I don't know -- libbpf_1 is in the latest release so I feel there should be a small window where people can keep using either libbpf_1 or libbpf to convert (it won't simplify backports to stable, but it'll simplify the life of anyone who wants to support both the latest stable and unstable). Once we have a release where libbpf is libbpf_1 we can then remove the old one (or now looking at aliases I guess making it throw)
With that said libbpf users is a small world, since Sascha is ok I'm not aware of anyone else who'd use it. As said previously, happy to remove it if asked again, I just wanted to make clear why I added it in the first place first.

Reply:
that is exactly what aliases.nix is doing. It will be reverted after the next release to a throw and then eventually is removed.